modules/navigation_3d/editor/navigation_obstacle_3d_editor_plugin.cpp

```diff
@@ -621,12 +622,12 @@ EditorPlugin::AfterGUIInput NavigationObstacle3DEditorPlugin::forward_3d_gui_inp
 			case MODE_DELETE: {
 				if (mb->get_button_index() == MouseButton::LEFT && mb->is_pressed()) {
 					int closest_idx = -1;
-					real_t closest_dist = 1e10;
+					real_t closest_dist_squared = 1e10;
```

Comment:
Keeping the existing behavior would be:

```suggestion
					real_t closest_dist_squared = 1e20;
```

There are more cases in this file.